templates/header.html

```diff
@@ -5,15 +5,15 @@
   <a id="logo" href="/"></a>
   <div id="actions">
     {{#metadata.preview}}
-    <a href="preview/" class="button hide-on-mobile"> 
+    <a href="{{urlPreview}}" class="button hide-on-mobile"> 
       <amp-img srcset="/img/ic_play_circle_filled_white_24dp_1x 1x, /img/ic_play_circle_filled_white_24dp_2x.png 2x" width="24" height="24" alt="preview"></amp-img>
-      <span>Show Preview</span>
+      <span>View Demo</span>
     </a>
     {{/metadata.preview}}
     {{#github}}
     <a href="{{github}}" class="button" target="_blank">
       <amp-img srcset="/img/GitHub-Mark-Light-32px.png 1x, /img/GitHub-Mark-Light-64px.png 2x" width="24" height="24" alt="Github" attribution="github" ></amp-img>
-      <span class="hide-on-mobile">View on Github</span>
+      <span class="hide-on-mobile">View Source</span>
```

Comment:
View Source sounds fine for me. We are also using a similar wording "View annotated Source" when we switch between preview and complete sample, so maybe we can change that to "View demo with code" or something similar?